src/store-enhancer.js

```diff
@@ -15,27 +37,39 @@ export default ({
       const vanillaState = {...state};
       delete vanillaState.router;
 
+      const newState = reducer(vanillaState, action);
+
+      // Support redux-loop
+      if (Array.isArray(newState)) {
```

Comment:
This is supported in IE9 and up, and considered ES5.  Do we call out in the README that ES5 environment is required?